cdap-ui/cypress/integration/nuxtour.spec.ts

```diff
@@ -17,7 +17,7 @@
 import * as Helpers from '../helpers';
 
 let headers = {};
-const controlCenterText = 'Control Center';
+const studioText = 'Pipeline Studio';
 
 // Assumes 5 step tour: Control Center, Pipeline Studio, Wrangler, Metadata, Hub
```

Comment:
Update the order of the steps in comment so that it starts with pipeline studio?